javascripts/govuk/analytics/analytics.js

```diff
@@ -13,6 +13,11 @@
       delete config.universalId
       this.trackers.push(new GOVUK.GoogleAnalyticsUniversalTracker(universalId, config))
     }
+    if (typeof config.govukTrackerGifUrl !== 'undefined') {
```

Comment:
If the calling application doesn't configure govukTrackerGifUrl, this tracker shouldn't be added to the stack